### DISCUSSION TITLE
4.0.1生成的dist文件夹，打开里面的index之后，显示 加载中

### DISCUSSION THREAD

**Discussion by Kanon1982** [imsyy/home#91] [Q&A]
用 yarn 生成之后的dist路径下，打开index.html，显示 ***的主页，加载中。
是我操作有问题吗？？？

**Comment by Kanon1982** [comment]
但是用 yarn dev 之后，显示正常

**Reply by imsyy** [reply]
不要直接打开，要么用 `vscode` 的 `live server`，要么上传到服务器并正确设置根目录后访问

**Reply by Kanon1982** [reply]
好的，谢谢大佬
